Tests/GeoLib/TestLineSegmentIntersect2d.cpp

```diff
@@ -50,7 +50,7 @@ class LineSegmentIntersect2dTest : public testing::Test
 	ac::gtest_reporter gtest_reporter;
 };
 
-#if not defined(_MSC_VER) && _MSC_VER < 1900
+#if !defined(_MSC_VER) || (_MSC_VER < 1900)
```

Comment:
`!defined(_MSC_VER)` disables tests most compilers, doesn't it?
I suggest `#if (!defined(_MSC_VER)) && _MSC_VER < 1900`